easybuild/easyconfigs/n/Ninja/Ninja-1.10.1-GCCcore-10.2.0.eb

```diff
@@ -14,18 +14,44 @@ checksums = ['a6b6f7ac360d4aabd54e299cc1d8fa7b234cd81b9401693da21221c62569a23e']
 
 builddependencies = [
     ('binutils', '2.35'),
+    ('CMake', '3.20.1'),
+    ('scikit-build', '0.15.0'),
+]
+
+dependencies = [
     ('Python', '3.8.6'),
 ]
 
 cmds_map = [('.*', "./configure.py --bootstrap")]
 
 files_to_copy = [(['ninja'], 'bin')]
 
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    ('ninja', '1.10.2.3', {
```

Comment:
As far as i could tell pypi "sources" aren't actually new code compared to the Ninja source tarball itself, just structured different + perhaps some autogenerated stuff (i think?), but, this approach worries me in 2 ways;
1. these version don't match with the easyconfig version, i think we should definitely do that
2. the python package also installs the bin/ninja binary (i think, at least, pip install ninja installs a bin/ninja). So i guess this is overwriting the one we just built?

so, perhaps we could opt to just use the pypi source? or, if possible, generate the python package from the original source?

Reply:
There are no 1.10.1.x versions of scikit-build, so can't "match" with Ninja 1.10.1...

I should look into the `bin/ninja` part, that may be a problem I overlooked indeed.

The sources for the `ninja` Python package are in https://github.com/scikit-build/ninja-python-distributions .
That mentions that the wheels include an actual `ninja` binary, but I'm not sure that's also the case when building from source...

Reply:
It looks like the `bin/ninja` that comes with the Python package is a light wrapper on top of the actual `ninja` binary (it's equivalent to the `ninja` function in https://github.com/scikit-build/ninja-python-distributions/blob/master/src/ninja/__init__.py)

Reply:
Seems it is? https://github.com/scikit-build/ninja-python-distributions/blob/450d8bb0dc375a536ee2a12eacca0d263dae3be2/CMakeLists.txt#L83-L102

Reply:
> It looks like the `bin/ninja` that comes with the Python package is a light wrapper on top of the actual `ninja` binary (it's equivalent to the `ninja` function in https://github.com/scikit-build/ninja-python-distributions/blob/master/src/ninja/__init__.py)

hm, it seems there are two `ninja`s (?) https://github.com/scikit-build/ninja-python-distributions/blob/450d8bb0dc375a536ee2a12eacca0d263dae3be2/CMakeLists.txt#L155-L156